frb_dart/lib/src/load/load.io.dart

```diff
@@ -4,8 +4,10 @@ import 'dart:io';
 /// agnostic dynamic library loading on native platforms
 ///
 /// see [#898](https://github.com/fzyzcjy/flutter_rust_bridge/pull/898)
-DynamicLibrary loadDylib(String path) => Platform.isIOS
+DynamicLibrary loadLibForFlutter(String path) => Platform.isIOS
     ? DynamicLibrary.process()
     : Platform.isMacOS && Abi.current() == Abi.macosX64
         ? DynamicLibrary.executable()
         : DynamicLibrary.open(path);
+DynamicLibrary loadLibForDart(String path) =>
+    Platform.isIOS ? DynamicLibrary.process() : DynamicLibrary.open(path);
```

Comment:
curious why macos test did pass before - Is it because we are missing some tests in CI?

Reply:
possibly, pure dart can't find static lib.

Reply:
the ci test only run on ubuntu

Reply:
At the beginning.
I don't agree, it's about build scripts. it's outside of frb, in other word, it's primarily for end users.
But now I think it might be necessary  -_-